src/main/window-manager.ts

```diff
@@ -81,10 +82,19 @@ export class WindowManager extends Singleton {
         this.splashWindow = null;
         app.dock?.hide(); // hide icon in dock (mac-os)
       });
+
+      this.mainWindow.webContents.on("did-fail-load", (_event, code, desc) => {
+        logger.error(`[WINDOW-MANAGER] Failed to load Main window`, code, desc);
```

Comment:
Winston does better with objects then raw values.
```suggestion
        logger.error(`[WINDOW-MANAGER]: Failed to load Main window`, { code, desc });
```